starknet_py/tests/e2e/docs/guide/test_using_existing_contracts.py

```diff
@@ -112,7 +110,10 @@ async def test_raw_call(account):
         selector=get_selector_from_name("balanceOf"),
         calldata=[account.address],
     )
+    # Pass the created call to Client.call_contract
     account_balance = await account.client.call_contract(call)
+
+    # Note that a Contract instance cannot be used here, since it needs ABI to generate the functions
```

Comment:
I think it'd be better to include a proper explanation in the docs themselves instead of comment in the snippet.

Reply:
I think it's kind of obvious, since there exists a subsection with the raw calling. Here it is close to the context.

Reply:
Okay, we can leave it as it is.